docs/topics/install.md

```diff
@@ -2,35 +2,44 @@
 
 This guide provides detailed information about installing and configuring the Brigade services.
 
+> Brigade is under highly active development. Day to day, the `master` branch is
+> changing. If you choose to build from source, you may prefer to build off of
+> a tagged release rather than master.
```

Comment:
That's a good line, and I think it might be highlighted by adding "because from day to day the images you might use to build your pipeline might be slightly out of sync with master, but in synch with tagged releases. "